lms/static/js/fixtures/instructor_dashboard/batchenrollment.html

```diff
@@ -0,0 +1,71 @@
+        <label>
+            Enter email addresses and/or usernames separated by new lines or commas.
+            You will not get notification for emails that bounce, so please double-check spelling.
+            <textarea rows="6" name="student-ids" placeholder="Email Addresses/Usernames" spellcheck="false"></textarea>
+        </label>
+        <input id="is_course_white_label" value="False" type="hidden">
+        <div class="enroll-option">
+            <label for="course-modes-list-selector" class="has-hint">Select a course mode:
+                <div class="hint auto-enroll-hint">
+                    <span class="hint-caret"></span>
+                    <p id="auto-enroll-tip">
+                        The users will be automatically enrolled in the <em>selected</em> course mode.
+                        <br><br>
+                        Selecting from course modes has no effect if 'Unenroll' is selected.
+                    </p>
+                </div>
+            </label>
+            <select id="course-modes-list-selector" name="course_mode" class="course-modes-list-selector">
+                <option value="honor">Honor Certificate</option>
+                <option value="verified">Verified Certificate</option>
+            </select>
+        </div>
+        <div class="enroll-option">
+            <label class="has-hint">
+                <input name="auto-enroll" id="auto-enroll" value="Auto-Enroll" checked="checked"
+                       aria-describedby="heading-batch-enrollment" type="checkbox">
+                <span class="label-text">Auto Enroll</span>
+                <div class="hint auto-enroll-hint">
+                    <span class="hint-caret"></span>
+                    <p id="auto-enroll-tip">
+                        If this option is <em>checked</em>, users who have not yet registered for Your Platform Name
+                        Here will be automatically enrolled.
+                        If this option is left <em>unchecked</em>, users who have not yet registered for Your Platform
+                        Name Here will not be enrolled, but will be allowed to enroll once they make an account.
+                        <br><br>
+                        Checking this box has no effect if 'Unenroll' is selected.
+                    </p>
+                </div>
+            </label>
+        </div>
+        <div class="enroll-option">
+            <label class="has-hint">
+                <input name="email-students" id="email-students" value="Notify-students-by-email" checked="checked"
+                       aria-describedby="heading-batch-enrollment" type="checkbox">
+                <span class="label-text">Notify users by email</span>
+                <div class="hint email-students-hint">
+                    <span class="hint-caret"></span>
+                    <p id="email-students-tip">
+                        If this option is <em>checked</em>, users will receive an email notification.
```

Comment:
If you select this option, learners receive an email message when they are successfully enrolled in your course.